main/tests/server/src/com/google/refine/tests/expr/functions/ToDateTests.java

```diff
@@ -26,16 +26,99 @@
  ******************************************************************************/
 package com.google.refine.tests.expr.functions;
 
+import com.google.refine.tests.RefineTest;
+import org.slf4j.LoggerFactory;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeTest;
 import org.testng.annotations.Test;
 
 import com.google.refine.expr.functions.ToDate;
 import com.google.refine.tests.util.TestUtils;
+import com.google.refine.expr.EvalError;
+import com.google.refine.grel.ControlFunctionRegistry;
+import com.google.refine.grel.Function;
+
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.Properties;
+
+public class ToDateTests extends RefineTest {
+
+    static Properties bindings;
+
+    @Override
+    @BeforeTest
+    public void init() {
+        logger = LoggerFactory.getLogger(this.getClass());
+    }
+
+    @Test
+    public void testUnixTime()
+    {
+        Assert.assertEquals((OffsetDateTime)invoke("toDate", 123456l, "unix"), OffsetDateTime.parse("1970-01-02T10:17:36+00:00"));
```

Comment:
I don't think it's necessary, but for consistency would be nice to use "OffsetDateTime.parse("1970-01-02T10:17:36"+ZoneOffset.UTC))" as per other tests